app/src/main/java/org/mifos/mobile/ui/fragments/RecentTransactionsFragment.kt

```diff
@@ -70,16 +59,20 @@ class RecentTransactionsFragment : BaseFragment(), RecentTransactionsView, OnRef
             inflater: LayoutInflater, container: ViewGroup?,
             savedInstanceState: Bundle?
     ): View? {
-        rootView = inflater.inflate(R.layout.fragment_recent_transactions, container, false)
-        ButterKnife.bind(this, rootView!!)
+         _binding = FragmentRecentTransactionsBinding.inflate(inflater,container,false)
+        errorLayoutBinding = ErrorLayoutBinding.inflate(inflater,container,false)
         recentTransactionsPresenter?.attachView(this)
-        sweetUIErrorHandler = SweetUIErrorHandler(activity, rootView)
+        sweetUIErrorHandler = SweetUIErrorHandler(activity, binding.root)
         showUserInterface()
         setToolbarTitle(getString(R.string.recent_transactions))
         if (savedInstanceState == null) {
             recentTransactionsPresenter?.loadRecentTransactions(false, 0)
         }
-        return rootView
+
+        errorLayoutBinding.btnTryAgain.setOnClickListener {
```

Comment:
A better way to put this up would be inside onViewCreated() because according to the lifecycle, onViewCreated() confirms that the view is created and now we can interact with the view elements.